src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        StringBuilder majorSB = new StringBuilder("Report for period");
+        String [] dateFromAR = dateFrom.split( " ");
+        String [] dateToAR = dateTo.split( " ");
+        String date1 = dateFromAR[dateFromAR.length -2].substring( 1);
+        String date2 = dateToAR[dateFromAR.length -2].substring(1,dateToAR[dateFromAR.length -2].length()-1);
+        majorSB.append(date1).append(" - ").append(date2).append(" ").append("\n");
+
+        final DateTimeFormatter pattern = DateTimeFormatter.ofPattern("dd-MM-yyyy");
```

Comment:
According to the checklist, the `DateTimeFormatter` should be a constant field. This will make the code easier to maintain and understand. Consider defining it as a private static final constant at the class level.